.github/workflows/carpenter_push.yaml

```diff
@@ -2,24 +2,44 @@ name: Carpentry Push
 on:
   push:
     branches:
-      - main
+      - "**"
 env:
-  IMAGE_NAME: arcaflow-plugin-template-python
+  IMAGE_NAME: arcaflow-plugin-uperf
```

Comment:
Is this change expected?

Reply:
It is not thank you for the catch. My original testing for this was on uperft and that was a missed Image name change from copy pasta. I will update